src/sync/peer-sync-controller.js

```diff
@@ -25,7 +30,8 @@ export class PeerSyncController {
   #roles
   /** @type {Record<Namespace, SyncCapability>} */
   #syncCapability = createNamespaceMap('unknown')
-  #isDataSyncEnabled = false
```

Comment:
Previously, sync could be in two possible states: data sync disabled and data sync enabled.

Now, there are three possible states: syncing nothing, only syncing initial namespaces, and syncing everything. I called these three states "namespace groups" and would love a better name.

Reply:
Re. naming, I think it's confusing to introduce the term "replicate" when we are using "sync" mainly here.

Also I think we should align the term we use in `PRESYNC_NAMESPACES` with the term `initial only`.

I suggest, although clunky and not the best:

- `type SyncEnabledState = 'none' | 'presync' | 'all'`
- `this.#syncEnabledState: SyncEnabledState = 'none'`
- `setSyncEnabledState(syncEnabledState)`

Reply:
Done in b8c5d7decd92d009dabdf0537ceb0f0a1ab941a7 and f427f649f112b2cbe56c190ce41f2e6ea530c3c8.